### PR TITLE
[Hub Generated] Review request for Microsoft.ContainerService to add version stable/2020-12-01

### PR DESCRIPTION
**Pull request by xiazhan**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/dev-containerservice-Microsoft.ContainerService-2020-12-01...master/).

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://msazure.visualstudio.com/One/_workitems/create/Scenario?templateId=d1c215ec-5d6a-4cb3-a193-95ddd0a1ba96&ownerId=85bb92fb-21fe-41ca-9309-47f2c5f57515). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json [hunk 1]
```diff
@@ -0,0 +1,3681 @@
...
+        },
+        "x-ms-examples": {
+          "Get Managed Cluster": {
+            "$ref": "./examples/ManagedClustersListClusterCredentialResult.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/listClusterMonitoringUserCredential": {
+      "post": {
```
Comment (fengzhou-msft) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json:332]: why use post method for the list APIs?

**Review by pilor — APPROVED** [review]
There is no delta between 2020-12-01 and 2020-11-01... not sure what is being reviewed here. no delta = no API changes

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json (hunk 1 above)
Comment (xiazhan) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json:332]: Not sure the initial reason, but its from existing published API. So should be fine.

**Comment by xiazhan** [comment]
Please hold and wait for changes from other feature owner to merge their changes

**Comment by xiazhan** [comment]
> There is no delta between 2020-12-01 and 2020-11-01... not sure what is being reviewed here. no delta = no API changes

Yes, we need wait for feature owners to merge their chagnes to the dev branch first.

**Comment by pilor** [comment]
> > There is no delta between 2020-12-01 and 2020-11-01... not sure what is being reviewed here. no delta = no API changes
> 
> Yes, we need wait for feature owners to merge their chagnes to the dev branch first.

Please don't open PRs until they are ready to be reviewed.  Or please make it very obvious in the PR title/description that it is not ready for review yet.

**Comment by xiazhan** [comment]
> > > There is no delta between 2020-12-01 and 2020-11-01... not sure what is being reviewed here. no delta = no API changes
> > 
> > 
> > Yes, we need wait for feature owners to merge their chagnes to the dev branch first.
> 
> Please don't open PRs until they are ready to be reviewed. Or please make it very obvious in the PR title/description that it is not ready for review yet.

Sorry, it is my first time doing the this work. Not familiar with the process.

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json [hunk 2]
```diff
@@ -0,0 +1,4077 @@
...
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeSpan"
+          },
+          "description": "Time slots on which upgrade is not allowed."
+        }
+      },
+      "description": "Default maintenance configuration properties."
+    },
+    "TimeInWeek": {
```
Comment (pilor) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-12-01/managedClusters.json:2576]: Please be explicit about what time zone is used here

**Review by pilor — APPROVED** [review]

**Pull request merged**